rfcs/0001-restructure.md

```diff
@@ -0,0 +1,192 @@
+# Restructure to improve composition
+
+## Proposal
+
+The existing Python language buildpacks will be broken down into a set of
+buildpacks that, by themselves, provide more limited functionality, but in
+composition, fill the same feature set that the language family already
+provides.
+
+The current set of buildpacks will be reorganized into the following:
+
+![Proposed Structure](/rfcs/assets/0001-proposed.png)
+
+* `cpython`
+  * provides: `cpython`
+  * requires: none
+
+* `pip`
+  * provides: `pip`
+  * requires: `cpython` during `build`
+
+* `pip-install`
+  * provides: `site-packages`
+  * requires: `cpython` and `pip` during `build`
+
+* `pipenv`
+  * provides: `pipenv`
+  * requires: `pip` and `cpython` during `build`
+
+* `pipenv-install`
+  * provides: `site-packages`
+  * requires: `cpython`, `pip`, and `pipenv` during `build`
```

Comment:
if "`pipenv-install buildpack will be responsible for calling pipenv install
directly, instead of delegating to pip install`", what is the role of `pip` here?
I was able to `pipenv install <module>` without having the `pip` tool on my path

Reply:
I was able to as well